cli-toolchain/quickstart.md

```diff
@@ -110,6 +111,8 @@ Do you want to use the default authentication and security configuration?
 
 #### Configuring auth with social providers
 
+Once you have your User Pool functioning, you can choose to add in more configurations such as Federation with Facebook, Google Sign-In, or Login with Amazon. You can also configure more advanced settings by selecting *Manual Configuration*.
```

Comment:
Once your User Pool is functioning, you can enable more configurations such as federation with Facebook, Google, or Login with Amazon. You can also configure more advanced settings by selecting *Manual Configuration*.